package.json

```diff
@@ -21,7 +21,8 @@
     "plugins": [
       "transform-react-jsx",
       "transform-object-rest-spread",
-      "transform-es2015-modules-commonjs"
+      "transform-es2015-modules-commonjs",
+      "transform-class-properties"
```

Comment:
Are you using this `class-properties` transform? I saw you were still binding methods.

Reply:
I forgot to remove it, i tried getting it work but for some reason it didnt want to.  Will remove.